### PR TITLE
Gc/get feature dimension

### PR DESCRIPTION
**Pull request by gcattan**
@toncho11 starting small with #48 :

`get_feature_dimension` disappeared from `qiskit-ml` in [this](https://github.com/Qiskit/qiskit-machine-learning/commit/c6ee17d77c48fae7b73dec4f5ad591e7474a1d26) commit.
I am reintroducing this method here.

### CODE REVIEW

**Review by qbarthelemy — CHANGES_REQUESTED** [review]
Can you add unitary tests for this new function?

File: pyriemann_qiskit/datasets/utils.py [hunk 1]
```diff
@@ -124,3 +124,47 @@ def get_linearly_separable_dataset():
...
+    # This code is licensed under the Apache License, Version 2.0. You may
+    # obtain a copy of this license in the LICENSE.txt file
+    # in the root directory of this source tree or at
+    # http://www.apache.org/licenses/LICENSE-2.0.
+    #
+    # Any modifications or derivative works of this code must retain this
+    # copyright notice, and modified files need to carry a notice indicating
+    # that they have been altered from the originals.
+    """
+    Check feature dimension of a given dataset.
```
Comment (qbarthelemy) [inline pyriemann_qiskit/datasets/utils.py:143]:
```suggestion
    Return the feature dimension of a given dataset.
```

File: pyriemann_qiskit/datasets/utils.py [hunk 2]
```diff
@@ -124,3 +124,47 @@ def get_linearly_separable_dataset():
...
+    if not isinstance(dataset, dict):
+        raise TypeError("Dataset is not formatted as a dict. Please check it.")
+
+    feature_dim = -1
+    for v in dataset.values():
+        if not isinstance(v, np.ndarray):
+            v = np.asarray(v)
+        return v.shape[1]
+
+    return feature_dim
```
Comment (qbarthelemy) [inline pyriemann_qiskit/datasets/utils.py:170]:
```suggestion
    for v in dataset.values():
        if not isinstance(v, np.ndarray):
            v = np.asarray(v)
        return v.shape[1]

    return -1
```

**Comment by gcattan** [comment]
Thanks for the review :)

**Pull request merged**